BUILD.gn

```diff
@@ -21,15 +25,19 @@ bundle_data("sparkle_resources") {
   outputs = [ "{{bundle_resources_dir}}/{{source_file_part}}" ]
 }
 
-config("sparkle_config") {
+# Configuration for building chrome/browser static library.
+config("sparkle_browser_build_config") {
   libs = [ "Sparkle.framework" ]
 
+  cflags = [ "-F" + rebase_path(root_out_dir, root_build_dir) ]
+}
+
+# Configuration for link sparkle into brave framework.
+config("sparkle_link_config") {
   ldflags = [
     "-Wl,-F" + rebase_path(root_out_dir, root_build_dir),
-  ]
-
-  cflags = [
-    "-F" + rebase_path(root_out_dir, root_build_dir),
+    "-Wl,-rpath,@loader_path/Frameworks",
```

Comment:
This is unsafe for release builds. In chromium these are all explicitly wrapped in `is_component_build` because otherwise any framework placed in those directories can be loaded which is a major security issue

Reply:
cc @diracdeltas

Reply:
By using install name, rpath isn't needed anymore.